src/test/java/net/uaznia/lukanus/hudson/plugins/gitparameter/UiAcceptanceTest.java

```diff
@@ -0,0 +1,95 @@
+package net.uaznia.lukanus.hudson.plugins.gitparameter;
+
+import hudson.model.FreeStyleProject;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
The fact that `commons-lang3` is exposed in the test harness is a bug that I fixed a few days ago in the most recent plugin POM. Better to depend on Commons Lang 2.x (which is delivered by core) or eliminate usage of Apache Commons in favor of the Java Platform's `String#isBlank`.